wcfsetup/install/files/lib/system/html/node/HtmlNodeUnfurlLink.class.php

```diff
@@ -63,4 +60,24 @@ private static function removeStyling(HtmlNodePlainLink $element): void
 
         $element->topLevelParent->appendChild($element->link);
     }
+
+    private static function lowercaseHostname(string $url): string
+    {
+        $uri = new Uri($url);
+        $uri->withHost(\mb_strtolower($uri->getHost()));
```

Comment:
The Uri class is immutable, thus this line is a noop. (generally speaking: `with*()` methods return a new object)

Reply:
Thanks for pointing this out.